### ISSUE TITLE
amp-sidebar: Facebook web view (IOS) bug

### ISSUE THREAD

**Issue by camelburrito** [ampproject/amphtml#3402]
amp-sidebar: Facebook web view (IOS) bug:
When you scroll the sidebar- the content below scrolls. We have already fixed this issue on IOS safari. We need to apply the same fix to FB webview

Code :
https://github.com/ampproject/amphtml/blob/master/extensions/amp-sidebar/0.1/amp-sidebar.js#L84

Test:

post www.ampproject.org on your FB. And then try clicking the link from FB IOS App. and open the sidebar using the hamburger menu. Now try scrolling the sidebar.

**Comment by rudygalfi** [comment]
Putting in Backlog.

**Comment by adelinamart** [comment]
@camelburrito is this still a bug?Thanks

**Comment by camelburrito** [comment]
i think @muxin solved this again -
@muxin can you test on FB webview and close?

**Comment by muxin** [comment]
Tested on facebook web view and this is not fixed yet.

**Comment by muxin** [comment]
Tried several approach but failed.
Adding a potential solution reference for future investigation: https://stackoverflow.com/questions/10238084/ios-safari-how-to-disable-overscroll-but-allow-scrollable-divs-to-scroll-norma

**Comment by aghassemi** [comment]
closing as stale
